app/views/comments/index.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.array!(@resource_comments) do |comment|
+  json.partial! 'comments/default', data: comment
+  json.extract! comment, :comment_on_id, :comment_on_type
+  json.partial! 'users/user', user: comment.user
```

Comment:
This `user` partial is already rendered in the `default` partial, won't this be a repetition?